kork/kork-plugins/src/main/java/com/netflix/spinnaker/config/Front50PluginsConfiguration.java

```diff
@@ -60,14 +65,36 @@ public class Front50PluginsConfiguration {
   private static final Logger log = LoggerFactory.getLogger(Front50PluginsConfiguration.class);
 
   @Bean
-  public static PluginOkHttpClientProvider pluginsOkHttpClient(Environment environment) {
+  public static PluginOkHttpClientProvider pluginsOkHttpClient(
+      Environment environment,
+      Provider<Registry> registry,
+      Retrofit2EncodeCorrectionInterceptor correctionInterceptor) {
     OkHttpClientConfigurationProperties okHttpClientProperties =
         Binder.get(environment)
             .bind("ok-http-client", Bindable.of(OkHttpClientConfigurationProperties.class))
             .orElse(new OkHttpClientConfigurationProperties());
 
+    OkHttpMetricsInterceptorProperties okHttpMetricsInterceptorProperties =
```

Comment:
I could use an education here.  Wouldn't we get the same behavior much more simply by adding an OkHttpMetricsInterceptorProperties argument to this method?